modules/library/main/src/main/java/org/geotools/filter/function/QuantileFunction.java

```diff
@@ -44,4 +46,14 @@ protected QuantileListVisitor getListVisitor() {
         // use a visitor to find the values in each bin
         return new QuantileListVisitor(getParameters().get(0), getClasses());
     }
+
+    @Override
+    protected boolean isPercentages() {
```

Comment:
Does not seem to be used? Reminder, a quantile function tries its best to create classes with the same count, but data might disagree and the even distribution might not happen. The percentage should report the actual class distribution, not the desired one.

Reply:
the method gets used by the abstract class. Regarding the uneven distribution tests have been added